.github/workflows/ubuntu.yml

```diff
@@ -32,3 +32,5 @@ jobs:
         run: cmake --build build -j=2
       - name: Test
         run: ctest --output-on-failure --test-dir build
+      - name: Run default benchmark
+        run: cd build && benchmarks/bench
```

Comment:
Should we keep these lines?

Reply:
Actually, let's keep it.

Reply:
Why not? It is useful to test that our benchmarks run... no?